testplan/web_ui/testing/src/Nav/NavEntry.js

```diff
@@ -22,6 +22,16 @@ import {
  */
 const NavEntry = (props) => {
   const badgeStyle = `${STATUS_CATEGORY[props.status]}Badge`;
+  const executionTime = (
+    props.displayTime && props.executionTime ? (
+      <i className={css(styles.entryIcon)} title='Execution time'>
+        <span className={css(styles[STATUS_CATEGORY[props.status]])}>
+          ({((new Date(props.executionTime.end)).getTime()
```

Comment:
I think it would be neater to calculate the timedelta separately and store in a variable before building the JSX content.

Reply:
I moved it to the outside in a function call.